tools/ci/travis/travis-ci.sh

```diff
@@ -223,8 +223,23 @@ CheckTypoTree () {
     else
       echo "NOT checking $1: $path (typo.prune)"
     fi
+    if [[ $path = 'configure' || $path = 'configure.ac' ]] ; then
+      touch CHECK_CONFIGURE
+    fi
   done)
   rm -f tmp-index
+  if [ -e CHECK_CONFIGURE ] ; then
+    rm -f CHECK_CONFIGURE
+    echo "configure or configure.ac altered in $1"
+    echo "Verifying that configure.ac generates configure"
+    git checkout "$1"
+    mv configure configure.ref
+    ./autogen
+    if ! diff -q configure configure.ref >/dev/null ; then
+      echo "configure.ac no longer generates configure"
```

Comment:
```suggestion
      echo "configure.ac no longer generates configure, please run ./autogen and commit"
```

Reply:
Hah - we need GitHub to run `check-typo` on suggestions :wink:

This is a good idea, I just had to reformat it for line-length.